test/fixtures/test-runner/never_ending_async.js

```diff
@@ -1,6 +1,4 @@
 const test = require('node:test');
 const { setTimeout } = require('timers/promises');
 
-test('never ending test', () => {
-  return setTimeout(100_000_000);
-});
\ No newline at end of file
+test('never ending test', () => setTimeout(100_000_000));
```

Comment:
Nit, shouldn't this just be a `() => new Promise(resolve => {})` or something similar? 

Currently it takes 100,000 seconds which is a bit more than a day which _should_ be plenty I guess.